src/clients/activity-client.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-import Client from 'src/shark/client'
+const Client = require('./base-client')
```

Comment:
ok, so why `require` instead of `import`?

Reply:
Because `import` belongs to a newer Javascript specification. 
I think node.js supports it in version 10.x but definitely not in 8.10, which is what we're using in Lamba.